apps/storybook/src/forms/RadioGroup.stories.tsx

```diff
@@ -125,7 +107,7 @@ export const ComplexExample1: Story = {
         <Flex className={styles.item} {...props} gap="xs">
           <RadioGroupItem
             endDecorator={
-              <Text color="fg.secondary" fontSize="sm">
+              <Text color="fg.secondary">
                 Displays a message as a bar at top of page
               </Text>
```

Comment:
let's change these to strings as well
```suggestion
              "Displays a message as a bar at top of page"
```